src/input.ts

```diff
@@ -246,3 +216,64 @@ export class Mouse {
     await this.click(x, y, { ...options, clickCount: 2 });
   }
 }
+
+const aliases = new Map<string, string[]>([
+  ['ShiftLeft', ['Shift']],
```

Comment:
These should come from when key !== code. I'd rather have some logic about the ordering in the key definitions map then the aliases here.

Reply:
Could you elaborate?